src/locales/metadata.json

```diff
@@ -6,6 +6,7 @@
     { "label": "Русский", "lang": "ru" },
     { "label": "繁體中文", "lang": "zh-TW" },
     { "label": "日本語", "lang": "ja" },
+    { "label": "Korean", "lang": "ko" },
```

Comment:
@olgierd-on-fire can you type "Korean" in Korean ?? I prefer to show every language in their own language instead of English :) It would be very friendly to Korean users I think !